Lib/test/test_bisect.py

```diff
@@ -126,6 +126,20 @@ def test_negative_lo(self):
         self.assertRaises(ValueError, mod.insort_left, [1, 2, 3], 5, -1, 3)
         self.assertRaises(ValueError, mod.insort_right, [1, 2, 3], 5, -1, 3)
 
+    def test_negative_hi(self):
+        # Issue 125889
+        def assertNoRiases(f, *arg, **kwargs):
+            try:
+                f(*arg, **kwargs)
+            except Exception:
+                self.fail('Unable to work negative hi argument')
```

Comment:
This should test the result as well.

Reply:
Can you be more specific? I don't seem to understand. Do you mean about the f function? ：（

Reply:
Yes, you should check the result of `f(*arg, **kwargs)`. You are just checking that it doesn't raise but since you changed the implementation, you should check that the result is still correct.